PKPD/inference/inference.py

```diff
@@ -158,7 +135,7 @@ def __init__(self, model: m.MultiOutputModel, times: np.ndarray, values: np.ndar
         self.objective_score = None
 
 
-    def find_optimal_parameter(self, initial_parameter:np.ndarray) -> None:
+    def find_optimal_parameter(self, initial_parameter:np.ndarray, number_of_iterations:int=5) -> None:
```

Comment:
Could you just document that the default of 5 iterations is (I think) arbitrary (in case we get hung up on it later)?